src/components/Icon/Expensicons.ts

```diff
@@ -110,6 +110,7 @@ import QuestionMark from '@assets/images/question-mark-circle.svg';
 import ReceiptSearch from '@assets/images/receipt-search.svg';
 import Receipt from '@assets/images/receipt.svg';
 import Rotate from '@assets/images/rotate-image.svg';
+import RotateLeft from '@assets/images/rotate-left.svg';
```

Comment:
Was removed in https://github.com/Expensify/App/pull/35019#discussion_r1482393049